# GPCHC数据
std_msgs/Header header # 时间戳
float64 heading # 航向角
float64 pitch
float64 roll
float64 gyo_x # rad/s
float64 gyo_y
float64 gyo_z
float64 acc_x # m/s^2
float64 acc_y
float64 acc_z
float64 lat # 经纬度 degree
float64 lon
float64 alt
int32 NSV1
int32 NSV2
int16 sys_status
int16 sat_status